### DISCUSSION TITLE
Modifying the 3D Model View during post-processing

### DISCUSSION THREAD

**Discussion by aknsj** [ansys/pyaedt#4499] [Q&A] [answered]
Hi, I am using ansys pyaedt for post processing of Icepak design. I have created a plot of temperature of the body using the following script which helps me a lot.

      def create_volume_temp_plot(plot_name,prefix_list):
          list = []
          for name in ipk.modeler.solid_names:
              for prefix in prefix_list:
                  if name.lower().startswith(prefix):
                      list.append(name)
      ipk.modeler.fit_all()
      temp_plot = ipk.post.create_fieldplot_volume(
          list,
          quantityName="Temperature",
          plot_name=plot_name
      )
      temp_plot.export_image()
      ipk.post.delete_field_plot(plot_name)

Now the problem is that I want to show only some of the solids in the plot of temperature, like the "fit selected" function in the toolbar, is there a corresponding pyaedt script for this?
Thank you very much for your reply.

![image](https://github.com/ansys/pyaedt/assets/163245306/384aed37-63b4-4fe2-879e-9f8ea770da24)

**Comment by lorenzovecchietti** [comment, marked as answer]
Hi, you can use the pyvista integration to view interactively or just export an image of your model. See the example here: https://github.com/ansys/pyaedt/discussions/4459

Let me know if this can work for you.

**Reply by aknsj** [reply]
Hi, I've read your script and if it's convenient, I'd like to know what script you used to achieve the effect of hiding the "Region" in the exported image. Thank you for your help.

**Reply by lorenzovecchietti** [reply]
You can use the `plot_air_objects` argument and set it to `False` or to `True`. If `False` the region object should not be exported.

**Reply by aknsj** [reply]
Thanks for your help, I found the plot_air_objects argument in the example you provided, but I want to export a temperature plot.
Currently I am using ipk.post.create_fieldplot_volume() function to export the temperature plot, but I didn't find the argument to hide the "Region" in this function, looking forward to your reply.
